code/datums/components/food/edible.dm

```diff
@@ -484,8 +484,9 @@ Behavior that's still missing from this component that original food items had t
 			food_taste_reaction = FOOD_DISLIKED
 		else if(foodtypes & H.dna.species.liked_food)
 			food_taste_reaction = FOOD_LIKED
+
 	if(food_flags & FOOD_SILVER_SPAWNED) // it's not real food
```

Comment:
would it be better to move this to below line 480, that way since jellypeople dont have toxic food, it would check other food types to see if they like it or not? Would also avoid checking for species like this.

Reply:
https://github.com/tgstation/tgstation/blob/ed3a59be53106b7fa1d90e4a6343596db0ef8848/code/datums/components/food/edible.dm#L480-L485

Basically making silver slime food count as disgusting BEFORE checking if the species likes that food, because Toxic food is neutral to Slimepeople, it will instead take the other food's flags into account (like meat, vegetable)

Reply:
it was revealed to me in a dream, I think the better solution however would be to make silver slime food a proper food flag.